Lib/uuid.py

```diff
@@ -455,8 +458,15 @@ def _netstat_getnode():
                 try:
                     words = line.rstrip().split()
                     word = words[i]
-                    if len(word) == 17 and word.count(b':') == 5:
-                        mac = int(word.replace(b':', b''), 16)
+                    if word.count(_mac_delim) == 5:
+                        if len(word) == 17:
+                            mac = int(word.replace(_mac_delim, b''), 16)
+                        elif len(word) < 17 and len(word) >= 11:
```

Comment:
I don't understand this change. Is it only needed on AIX? I would prefer to keep ==17 test on other platforms.

Reply:
As I do not have access to other platforms I was trying to write something that was generic. I'll come up with something else.